### PR TITLE
Adding table of contents

### PR DESCRIPTION
**Pull request by fariddinar**
Organizing all the chapters and their associated notebooks for easier navigation.

### CODE REVIEW

**Comment by merveenoyan** [comment]
@fariddinar no need, when the course is rendered it will automatically be there due to `_toctree`

**Comment by merveenoyan** [comment]
@fariddinar thank you! I think what would be better would be to open notebooks in Colabs and putting a share link such that people can directly try them out

**Comment by fariddinar** [comment]
@merveenoyan  Certainly! It would be easier for the learners. For instance we have 29 notebooks, and I'm willing to collaborate on some of them in Colab.

**Review by ratan — COMMENTED** [review]
good effort

**Comment by merveenoyan** [comment]
@fariddinar we are very close to shipping the course. would you be able to do this soon or I can contribute to your branch as well?

**Review by ATaylorAerospace — APPROVED** [review]
Thanks for your efforts @fariddinar

File: chapters/en/Unit 0 - Welcome/TableOfContents.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+|--------------------------------------------------------|----------------------------------------------------------|----------|
+| Unit 4 - Multimodal Models                             | [ClipCrop](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%204%20-%20Multimodal%20Models/ClipCrop.ipynb)                                          | [ClipCrop](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%204%20-%20Multimodal%20Models/ClipCrop.ipynb)    |
+|                                                        | [Clip_finetune](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%204%20-%20Multimodal%20Models/Clip_finetune.ipynb)                                     | [Clip_finetune](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%204%20-%20Multimodal%20Models/Clip_finetune.ipynb)    |
```
Comment (merveenoyan) [inline chapters/en/Unit 0 - Welcome/TableOfContents.mdx:28]:
```suggestion
|                                                        | [Fine-tuning CLIP](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%204%20-%20Multimodal%20Models/Clip_finetune.ipynb)                                     | [Fine-tuning CLIP](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%204%20-%20Multimodal%20Models/Clip_finetune.ipynb)    |
```

File: chapters/en/Unit 0 - Welcome/TableOfContents.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+|                                                        | [Image classification with CLIP](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%204%20-%20Multimodal%20Models/CLIP%20and%20relatives/Image%20classification%20with%20CLIP.ipynb)                   | [Image classification with CLIP](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%204%20-%20Multimodal%20Models/CLIP%20and%20relatives/Image%20classification%20with%20CLIP.ipynb)    |
+|                                                        | [Image_retrieval_with_prompts](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%204%20-%20Multimodal%20Models/CLIP%20and%20relatives/Image_retrieval_with_prompts.ipynb)                      | [Image_retrieval_with_prompts](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%204%20-%20Multimodal%20Models/CLIP%20and%20relatives/Image_retrieval_with_prompts.ipynb)    |
```
Comment (merveenoyan) [inline chapters/en/Unit 0 - Welcome/TableOfContents.mdx:31]:
```suggestion
|                                                        | [Image Retrieval with Prompts](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%204%20-%20Multimodal%20Models/CLIP%20and%20relatives/Image_retrieval_with_prompts.ipynb)                      | [Image Retrieval with Prompts](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%204%20-%20Multimodal%20Models/CLIP%20and%20relatives/Image_retrieval_with_prompts.ipynb)    |
```

File: chapters/en/Unit 0 - Welcome/TableOfContents.mdx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+|                                                        | [Image_similarity](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%204%20-%20Multimodal%20Models/CLIP%20and%20relatives/Image_similarity.ipynb)                                  | [Image_similarity](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%204%20-%20Multimodal%20Models/CLIP%20and%20relatives/Image_similarity.ipynb)    |
```
Comment (merveenoyan) [inline chapters/en/Unit 0 - Welcome/TableOfContents.mdx:32]:
```suggestion
|                                                        | [Image Similarity](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%204%20-%20Multimodal%20Models/CLIP%20and%20relatives/Image_similarity.ipynb)                                  | [Image Similarity](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%204%20-%20Multimodal%20Models/CLIP%20and%20relatives/Image_similarity.ipynb)    |
```

File: chapters/en/Unit 0 - Welcome/TableOfContents.mdx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+|--------------------------------------------------------|----------------------------------------------------------|----------|
+| Unit 5 - Generative Models                             | No Notebook                                              | No Colab |
+|--------------------------------------------------------|----------------------------------------------------------|----------|
+| Unit 6 - Basic CV Tasks                                | [Fine_tune_SAM_on_Custom_Dataset](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%206%20-%20Basic%20CV%20Tasks/Fine_tune_SAM_(Segment_Anything_Model)_on_Custom_Dataset.ipynb)                   | [Fine_tune_SAM_on_Custom_Dataset](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%206%20-%20Basic%20CV%20Tasks/Fine_tune_SAM_(Segment_Anything_Model)_on_Custom_Dataset.ipynb)    |
```
Comment (merveenoyan) [inline chapters/en/Unit 0 - Welcome/TableOfContents.mdx:36]:
```suggestion
| Unit 6 - Basic CV Tasks                                | [Fine-tune SAM on Custom Dataset](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%206%20-%20Basic%20CV%20Tasks/Fine_tune_SAM_(Segment_Anything_Model)_on_Custom_Dataset.ipynb)                   | [Fine-tune SAM on Custom Dataset](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%206%20-%20Basic%20CV%20Tasks/Fine_tune_SAM_(Segment_Anything_Model)_on_Custom_Dataset.ipynb)    |
```

File: chapters/en/Unit 0 - Welcome/TableOfContents.mdx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+|--------------------------------------------------------|----------------------------------------------------------|----------|
+| Unit 7 - Video and Video Processing                    | No Notebook                                              | No Colab |
+|--------------------------------------------------------|----------------------------------------------------------|----------|
+| Unit 8 - 3D Vision, Scene Rendering, and Reconstruction| No Notebook                                              | No Colab |
+|--------------------------------------------------------|----------------------------------------------------------|----------|
+| Unit 9 - Model Optimization                            | [edge_tpu](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/edge_tpu.ipynb) | [edge_tpu](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/edge_tpu.ipynb) |
```
Comment (merveenoyan) [inline chapters/en/Unit 0 - Welcome/TableOfContents.mdx:42]:
```suggestion
| Unit 9 - Model Optimization                            | [Edge TPU](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/edge_tpu.ipynb) | [Edge TPU](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/edge_tpu.ipynb) |
```

File: chapters/en/Unit 0 - Welcome/TableOfContents.mdx [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+|													     | [onnx](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/onnx.ipynb)         | [onnx](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/onnx.ipynb)         |
```
Comment (merveenoyan) [inline chapters/en/Unit 0 - Welcome/TableOfContents.mdx:43]:
```suggestion
|													     | [ONNX](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/onnx.ipynb)         | [ONNX](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/onnx.ipynb)         |
```

File: chapters/en/Unit 0 - Welcome/TableOfContents.mdx [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+|													     | [openvino](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/openvino.ipynb) | [openvino](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/openvino.ipynb) |
```
Comment (merveenoyan) [inline chapters/en/Unit 0 - Welcome/TableOfContents.mdx:44]:
```suggestion
|													     | [OpenVINO](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/openvino.ipynb) | [OpenVINO](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/openvino.ipynb) |
```

File: chapters/en/Unit 0 - Welcome/TableOfContents.mdx [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+|													     | [optimum](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/optimum.ipynb)   | [optimum](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/optimum.ipynb)   |
```
Comment (merveenoyan) [inline chapters/en/Unit 0 - Welcome/TableOfContents.mdx:45]:
```suggestion
|													     | [Optimum](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/optimum.ipynb)   | [Optimum](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/optimum.ipynb)   |
```

File: chapters/en/Unit 0 - Welcome/TableOfContents.mdx [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+|													     | [tensorrt](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/tensorrt.ipynb) | [tensorrt](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/tensorrt.ipynb) |
```
Comment (merveenoyan) [inline chapters/en/Unit 0 - Welcome/TableOfContents.mdx:46]:
```suggestion
|													     | [TensorRT](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/tensorrt.ipynb) | [TensorRT](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/tensorrt.ipynb) |
```

File: chapters/en/Unit 0 - Welcome/TableOfContents.mdx [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+|													     | [tmo](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/tmo.ipynb)           | [tmo](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/tmo.ipynb)           |
```
Comment (merveenoyan) [inline chapters/en/Unit 0 - Welcome/TableOfContents.mdx:47]:
```suggestion
|													     | [TMO](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/tmo.ipynb)           | [TMO](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/tmo.ipynb)           |
```

File: chapters/en/Unit 0 - Welcome/TableOfContents.mdx [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+|													     | [torch](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/torch.ipynb)       | [torch](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/torch.ipynb)       |
```
Comment (merveenoyan) [inline chapters/en/Unit 0 - Welcome/TableOfContents.mdx:48]:
```suggestion
|													     | [Torch](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/torch.ipynb)       | [Torch](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%209%20-%20Model%20Optimization/torch.ipynb)       |
```

File: chapters/en/Unit 0 - Welcome/TableOfContents.mdx [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+|--------------------------------------------------------|----------------------------------------------------------|----------|
+| Unit 10 - Synthetic Data Creation                      | [OWLV2_labeled_image_dataset_with_annotations](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/OWLV2_labeled_image_dataset_with_annotations.ipynb)     | [OWLV2_labeled_image_dataset_with_annotations](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/OWLV2_labeled_image_dataset_with_annotations.ipynb)    |
```
Comment (merveenoyan) [inline chapters/en/Unit 0 - Welcome/TableOfContents.mdx:50]:
```suggestion
| Unit 10 - Synthetic Data Creation                      | [Dataset Labeling with OWLv2](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/OWLV2_labeled_image_dataset_with_annotations.ipynb)     | [Dataset Labeling with OWLv2](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/OWLV2_labeled_image_dataset_with_annotations.ipynb)    |
```

File: chapters/en/Unit 0 - Welcome/TableOfContents.mdx [hunk 13]
```diff
@@ -0,0 +1,60 @@
...
+|													     | [Synthetic_lung_images_hf_course](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/Synthetic_lung_images_hf_course.ipynb)                   | [Synthetic_lung_images_hf_course](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/Synthetic_lung_images_hf_course.ipynb)    |
```
Comment (merveenoyan) [inline chapters/en/Unit 0 - Welcome/TableOfContents.mdx:51]:
```suggestion
|													     | [Generating Synthetic Lung Images](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/Synthetic_lung_images_hf_course.ipynb)                   | [Generating Synthetic Lung Images](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/Synthetic_lung_images_hf_course.ipynb)    |
```

File: chapters/en/Unit 0 - Welcome/TableOfContents.mdx [hunk 14]
```diff
@@ -0,0 +1,60 @@
...
+|													     | [blenderproc_examples](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/blenderproc_examples.ipynb)                              | [blenderproc_examples](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/blenderproc_examples.ipynb)    |
```
Comment (merveenoyan) [inline chapters/en/Unit 0 - Welcome/TableOfContents.mdx:52]:
```suggestion
|													     | [BlenderProc Examples](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/blenderproc_examples.ipynb)                              | [BlenderProc Examples](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/blenderproc_examples.ipynb)    |
```

File: chapters/en/Unit 0 - Welcome/TableOfContents.mdx [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+|													     | [image_labeling_BLIP_2](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/image_labeling_BLIP_2.ipynb)                            | [image_labeling_BLIP_2](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/image_labeling_BLIP_2.ipynb)    |
```
Comment (merveenoyan) [inline chapters/en/Unit 0 - Welcome/TableOfContents.mdx:53]:
```suggestion
|													     | [Image Labeling with BLIP-2](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/image_labeling_BLIP_2.ipynb)                            | [Image Labeling with BLIP-2](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/image_labeling_BLIP_2.ipynb)    |
```

File: chapters/en/Unit 0 - Welcome/TableOfContents.mdx [hunk 16]
```diff
@@ -0,0 +1,60 @@
...
+|													     | [synthetic_data_creation_sdxl_turbo](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/synthetic_data_creation_sdxl_turbo.ipynb)               | synthetic_data_creation_sdxl_turbo](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/synthetic_data_creation_sdxl_turbo.ipynb)    |
```
Comment (merveenoyan) [inline chapters/en/Unit 0 - Welcome/TableOfContents.mdx:54]:
```suggestion
|													     | [Synthetic Data Creation with SDXL Turbo](https://github.com/johko/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/synthetic_data_creation_sdxl_turbo.ipynb)  |             [Synthetic Data Creation with SDXL Turbo](https://colab.research.google.com/github/fariddinar/computer-vision-course/blob/main/notebooks/Unit%2010%20-%20Synthetic%20Data%20Creation/synthetic_data_creation_sdxl_turbo.ipynb)    |
```

File: chapters/en/Unit 0 - Welcome/TableOfContents.mdx [hunk 17]
```diff
@@ -0,0 +1,60 @@
+# Table of Contents
```
Comment (merveenoyan) [inline chapters/en/Unit 0 - Welcome/TableOfContents.mdx:1]:
```suggestion
# Table of Contents for Notebooks
```
also maybe reflect this change for the filename as well. we already have a toctree that renders as sidebar in the course itself

**Review by fariddinar — COMMENTED** [review]
Just submitted. Hope it helps!

**Comment by merveenoyan** [comment]
@fariddinar can you add this chapter under Welcome section in `_toctree.yml`? - Unit 0 - Welcome/TableOfContents

**Comment by fariddinar** [comment]
@merveenoyan I Just updated _toctree.yml by adding TableOfContents :)

**Pull request merged**